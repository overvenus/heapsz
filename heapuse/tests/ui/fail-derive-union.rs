use heapuse_derive::Heap;

#[derive(Heap)]
union Union {
    f1: u32,
    f2: f32,
}

fn main() {}
