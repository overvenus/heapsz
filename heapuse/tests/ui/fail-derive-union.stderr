error: `Heap` can not be derived for a union
 --> tests/ui/fail-derive-union.rs:4:1
  |
4 | / union Union {
5 | |     f1: u32,
6 | |     f2: f32,
7 | | }
  | |_^
